xunit-test/NearCacheTest/NearCacheEvictionTest.cpp

```diff
@@ -59,6 +57,8 @@ TEST_F(NearCacheEvictionTest, EvictionOnFullNearCacheTest) {
     auto stats0 = cache.stats();
     cache.get("key1");
     cache.put("key1", "value1");
+    // Sleep some time. This avoids that the events related to the prev put arrives after the next get
+    std::this_thread::sleep_for(milliseconds(50));
```

Comment:
instead of waiting for a fixed amount of time, is it possible to wait until some condition ? e.g. consume all events ?

Reply:
I know it's ugly, but there's no easy condition to check, unless we want to instrument the near cache implementation with statistics